stories/SQFormDialog.stories.tsx

```diff
@@ -125,7 +159,7 @@ export const WithDatePickers: SQFormDialogWithDatePickersStory = (args) => {
   );
 };
 WithDatePickers.args = {
-  ...defaultArgs,
+  //...defaultArgs,
```

Comment:
[question] Was there a particular reason we wanted this commented out instead of removed like above and below it?